### PR TITLE
[DVCSMP-1588] changes to account for bad configure method for ge link.

### PR DESCRIPTION
**Pull request by workingmonk**
to account for users with bad config, hitting refresh should solve it for them.

cc @tpmanley @tylerlange

### CODE REVIEW

**Comment by tpmanley** [comment]
:+1:

File: devicetypes/smartthings/ge-link-bulb.src/ge-link-bulb.groovy [hunk 1]
```diff
@@ -270,109 +188,56 @@ def updated() {
...
-    state.trigger = "setLevel"
-    state.lvl = "${level}"
-
     if (dimRate && (state?.rate != null)) {
-    	cmds << "st cmd 0x${device.deviceNetworkId} 1 8 4 {${level} ${state.rate}}"
+        zigbee.setLevel(value, convertRateValue(state.rate))
     }
     else {
-    	cmds << "st cmd 0x${device.deviceNetworkId} 1 8 4 {${level} 1500}"
+        zigbee.setLevel(value, 15)
```
Comment (tpmanley) [inline devicetypes/smartthings/ge-link-bulb.src/ge-link-bulb.groovy]: The 1500 is hex, so decimal should be 21 or 20 to make it even

**Comment by tylerlange** [comment]
Can someone add in the fingerprint for the new one as well?

**Comment by tylerlange** [comment]
@workingmonk Here's what I had, feel free to modify:

 fingerprint profileId: "0104", inClusters: "0000,0003,0004,0005,0006,0008,1000", outClusters: "0019", manufacturer: "GE", model: "Daylight", deviceJoinName: "GE Link Bulb"

**Comment by workingmonk** [comment]
@tylerlange did the manufacturer name change for GE? the current GE link bulb says that the manufacturer name is 'GE_Appliances'. Do you know when we'll get the device? And should we wait to test this DTH with that before adding it here?

**Comment by workingmonk** [comment]
commits squashed, waiting for Tom's approval

**Comment by tpmanley** [comment]
:+1:

**Pull request merged**